rotkehlchen/data/importer.py

```diff
@@ -469,3 +480,269 @@ def import_cryptocom_csv(self, filepath: Path) -> Tuple[bool, str]:
+        elif entry_type in ('Withdrawal', 'Wire Withdrawal', 'ACH Withdrawal'):
+            asset_movement = AssetMovement(
+                location=Location.BLOCKFI,
+                category=AssetMovementCategory.WITHDRAWAL,
+                address=None,
+                transaction_id=None,
+                timestamp=timestamp,
+                asset=asset,
+                amount=amount,
+                fee=fee,
+                fee_asset=fee_asset,
+                link='',
+            )
+            self.db.add_asset_movements([asset_movement])
+        elif entry_type == 'Withdrawal Fee':
+            action = LedgerAction(
+                identifier=0,  # whatever is not used at insertion
+                timestamp=timestamp,
+                action_type=LedgerActionType.EXPENSE,
+                location=Location.BLOCKFI,
+                amount=amount,
+                asset=asset,
+                rate=None,
+                rate_asset=None,
+                link=None,
+                notes=f'{entry_type} from BlockFi',
+            )
+            self.db_ledger.add_ledger_action(action)
+        elif entry_type in ('Interest Payment', 'Bonus Payment', 'Referral Bonus'):
+            action = LedgerAction(
+                identifier=0,  # whatever is not used at insertion
+                timestamp=timestamp,
+                action_type=LedgerActionType.INCOME,
+                location=Location.BLOCKFI,
+                amount=amount,
+                asset=asset,
+                rate=None,
+                rate_asset=None,
+                link=None,
+                notes=f'{entry_type} from BlockFi',
+            )
+            self.db_ledger.add_ledger_action(action)
+        elif entry_type == 'Trade':
+            pass
+        else:
+            raise UnsupportedBlockFiEntry(f'Unsuported entry {entry_type}. Data: {csv_row}')
+
+    def import_blockfi_transactions_csv(self, filepath: Path) -> Tuple[bool, str]:
+        """
+        Information for the values that the columns can have has been obtained from
+        https://github.com/BittyTax/BittyTax/blob/06794f51223398759852d6853bc7112ffb96129a/bittytax/conv/parsers/blockfi.py#L67
+        """
+        with open(filepath, 'r', encoding='utf-8-sig') as csvfile:
+            data = csv.DictReader(csvfile)
+            for row in data:
+                try:
+                    self._consume_blockfi_entry(row)
+                except UnknownAsset as e:
+                    self.db.msg_aggregator.add_warning(
+                        f'During BlockFi CSV import found action with unknown '
+                        f'asset {e.asset_name}. Ignoring entry',
+                    )
+                    continue
+                except DeserializationError as e:
+                    self.db.msg_aggregator.add_warning(
+                        f'Error during BlockFi CSV import deserialization. '
+                        f'Error was {str(e)}. Ignoring entry',
```

Comment:
"Error was XXX" is not needed and is going to read wrong for all errors.

```suggestion
                        f'Deserialization error during BlockFi CSV import. '
                        f'{str(e)}. Ignoring entry',
```